src/scripts/main.js

```diff
@@ -1 +1,56 @@
 'use strict';
+
+function showNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type;
+  notification.textContent = message;
+  document.body.appendChild(notification);
+}
+
+const firstPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('click', () => {
+    clearTimeout(timer);
+    resolve('First promise was resolved');
+  });
+
+  const timer = setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => showNotification(message, 'success'))
+  .catch((error) => showNotification(error, 'error'));
```

Comment:
When handling the error in the 'catch' block, consider using 'error.message' instead of passing the entire error object to 'showNotification'. This will ensure that only the error message is displayed.